src/test/resources/features/Sessions.feature

```diff
@@ -0,0 +1,315 @@
+Feature: Session template based tests
+  As a user, I should be able to view the Prison sessions based on session restrictions within the Prison establishments
+# All the Test Data is pre-set in Admin service & via NOMIS ensure the Test Data is up-to-date with regards to session restrictions(in Admin service) within each establishment. All the tests are using Drake hall & Bristol establishments.
+
+  @suite
+  Scenario Outline: Open session - User able to book an Open session for all Prisoners within the establishment
+    Given I navigate to VSIP service
+    And Im on "HMPPS Digital Services - Sign in" page
+    And I enter "<userName>"
+    And I enter the "<password>"
+    And click on submit button
+    Then Im on "Manage prison visits - Manage prison visits" page
+    And I select change establishment
+    And I change the establishmnet to Bristol
+    And click on continue button
+    And I click on Book a visit option
+    And Im on "Manage prison visits - Search for a prisoner" page
+    When I enter "<prisonNumber>" to search for a prison
+    And click on search button
+    Then I choose prison from search results
+    Then Im on "Manage prison visits - Vsip_prisoner12, Do Not Use" page
+    And I click on Book a visit button
+    Then Im on "Manage prison visits - Select visitors from the prisoner’s approved visitor list" page
+    And I select a visitr form the list from Bristol establishment
+    And click on continue button
+    Then Im on "Manage prison visits - Select date and time of visit" page
+    And I select time slot
```

Comment:
Are we not selecting the first available time slot? How are we ensuring that no closed sessions are being shown for that prisoner? That was the main aim of this test.

Reply:
same as above covered in Vb 3308 additional tests